socorro/external/boto/crashstorage.py

```diff
@@ -292,6 +292,20 @@ class TelemetryBotoS3CrashStorage(BotoS3CrashStorage):
         reference_value_from='resource.elasticsearch',
     )
 
+    required_config.add_option(
+        'keybuilder_class',
+        default=(
+            'socorro.external.boto.connection_context'
+            '.SimpleDatePrefixKeyBuilder'
+        ),
+        doc=(
+            'fully qualified dotted Python classname to handle building s3 '
+            'pseudo-filenames'
+        ),
+        from_string_converter=class_converter,
+        reference_value_from='resource.boto',
```

Comment:
I suspect you're right. I might have rushed this. 

Now that you mention it, it's true that we probably never want to have anything other than SimpleDatePrefixKeyBuilder so it could be moved from configuration to hardcoded in `__init__`.